src/20250421_Multi_GHOGnosisLaunch/utils/GHOLaunchConstants.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title GHOLaunchConstants
+ * @notice Library containing all constants used across the GHO Gnosis Launch proposal
+ */
+library GHOLaunchConstants {
+  // Common Addresses
+  address internal constant RISK_COUNCIL = 0x8513e6F37dBc52De87b166980Fa3F50639694B60;
+
+  // CCIP Rate Limits
+  uint128 internal constant CCIP_RATE_LIMIT_CAPACITY = 1_500_000e18;
+  uint128 internal constant CCIP_RATE_LIMIT_REFILL_RATE = 300e18;
+  uint128 internal constant CCIP_BUCKET_CAPACITY = 40_000_000e18;
+
+  // Arbitrum Addresses
+  address internal constant ARB_TOKEN_ADMIN_REGISTRY = 0x39AE1032cF4B334a1Ed41cdD0833bdD7c7E7751E;
+  address internal constant ARB_CCIP_ROUTER = 0x141fa059441E0ca23ce184B6A78bafD2A517DdE8;
+
+  // Arbitrum ON_RAMPs
+  address internal constant ARB_GNO_ON_RAMP = 0xc7d6B885d8A4286E6311F79227430b7862311cd3;
+  address internal constant ARB_ETH_ON_RAMP = 0x67761742ac8A21Ec4D76CA18cbd701e5A6F3Bef3;
```

Comment:
why are the addresses that are not of the path chain <-> GNO needed?
I think they are only used for the tests, but is it needed to test all the other paths, instead of only the new one? (same for all the chains)